### PR TITLE
fix move to finish lua script

### PR DESCRIPTION
**Pull request by adiholden**

### CODE REVIEW

File: tests/test_worker.ts [hunk 1]
```diff
@@ -666,12 +666,11 @@ describe('workers', function () {
       };
     });
 
+
+    await Promise.all([...normalPriority,...mediumPriority,...highPriority]);
     const worker = new Worker(queueName, processor, { connection });
     await worker.waitUntilReady();
 
-    // wait for all jobs to enter the queue and then start processing
```
Comment (chakaz) [inline tests/test_worker.ts:672]: Revive this comment maybe?

File: tests/test_worker.ts [hunk 2]
```diff
@@ -710,6 +709,41 @@ describe('workers', function () {
     await worker.close();
   });
 
+
+  describe('when prioritized job is added while processing last active job', () => {
+    it('should process prioritized job whithout delay', async function () {
```
Comment (chakaz) [inline tests/test_worker.ts]: without (typo)

File: tests/test_worker.ts [hunk 3]
```diff
@@ -710,6 +709,41 @@ describe('workers', function () {
...
+      this.timeout(1000);
```
Comment (chakaz) [inline tests/test_worker.ts:715]: why?

File: tests/test_worker.ts [hunk 4]
```diff
@@ -710,6 +709,41 @@ describe('workers', function () {
...
+            reject(err);
+          }
+
+          if (++counter === 2) {
+            resolve();
+          }
+        };
+      });
+
+      const worker = new Worker(queueName, processor, { connection });
```
Comment (chakaz) [inline tests/test_worker.ts:737]: Can't it be that you're accessing `processor` before it is assigned?

File: tests/test_worker.ts (hunk 3 above)
Comment (adiholden) [reply tests/test_worker.ts:715]: I copied this test from https://github.com/taskforcesh/bullmq/pull/2176
Did not ask questions about it

File: tests/test_worker.ts (hunk 2 above)
Comment (adiholden) [reply tests/test_worker.ts]: Its a typo from the original code but sure I can fix it in our branch

File: tests/test_worker.ts (hunk 4 above)
Comment (adiholden) [reply tests/test_worker.ts:737]: same comment as above

Comment (chakaz) [reply tests/test_worker.ts:737]: Oh, I didn't realize this is copied from their PR, sorry! :)

**Review by chakaz — APPROVED** [review]

**Pull request merged**